Dockerfile

```diff
@@ -1,28 +1,26 @@
-FROM mongo:2.6
+FROM alpine:3.4
```

Comment:
@forhau Maybe you can just use the node base and that way you do not need to install dnsmasq and serf.